libs/src/sdk/api/chats/ChatsApi.ts

```diff
@@ -261,7 +265,7 @@ export class ChatsApi
           sharedSecret,
           base64.decode(m.message)
         ).catch((e) => {
-          console.error(
+          this.logger.error(
             "[audius-sdk]: Error: Couldn't decrypt chat message",
```

Comment:
If you're feeling up for it, would be great to remove the `[audius-sdk]` prefixes for these now